src/FixedDataTableCellGroup.js

```diff
@@ -73,21 +136,40 @@ class FixedDataTableCellGroupImpl extends React.Component {
     this._initialRender = false;
   }
 
+  /**
+   * Returns Object consisting of keys and widths of the columns in the current cell group.
+   * @returns {{keys: [], widths: []}}
+   */
+  getCellGroupWidth = () => {
```

Comment:
Is this needed anymore?

Reply:
yes, It is used in ReorderHandle to get cell group width